site/en/docs/extensions/mv3/devtools/index.md

```diff
@@ -436,69 +443,53 @@ var backgroundPageConnection = chrome.runtime.connect({
 
 Browse the source of these DevTools extension examples:
 
-- [Polymer Devtools Extension][37] - uses many helpers running in the host page to query DOM/JS
-  state to send back to the custom panel.
-- [React DevTools Extension][38] - Uses a submodule of Blink to reuse DevTools UI components.
-- [Ember Inspector][39] - Shared extension core with adapters for both Chrome and Firefox.
-- [Coquette-inspect][40] - A clean React-based extension with a debugging agent injected into the
-  host page.
-- our [DevTools Extension Gallery][41] and [Sample Extensions][42] have more worthwhile extensions to
-  install, try out, and learn from.
+- [Polymer Devtools Extension][gh-polymer-dt] - uses many helpers running in the host page to query
+  DOM/JS state to send back to the custom panel.
+- [React DevTools Extension][gh-react-dt] - Uses a submodule of Blink to reuse DevTools UI
```

Comment:
```suggestion
- [React DevTools Extension][gh-react-dt] - Uses a submodule of the renderer to reuse DevTools UI
```

Reply:
This may not be the correct rewrite. Regardless, Chromium code names should be in external documents.